ccmbioinfo/lfs
# CARDS-1668: PROMs: relatedSubjects not set for forms imported from torch

To test:
- checkout the `torch-mock-server` branch
- `cd Utilities/Development`
- `npm install body-parser express`
- `node ./mock_torch.js  --appointment-time-hours-from-now=24` (leave it running in a separate tab)
- checkout this branch, build, start with `-P cards4proms --dev`
- go to `http://localhost:8080/system/console/configMgr`
- look for the `PROMs import` configuration, click on it to add a new config, set the *Name* to `mock`, *Endpoint URL* to `http://localhost:8011/`, save
- open `http://localhost:8080/Subjects.importTorch.json?config=mock`
- either in `http://localhost:8080/bin/browser.html/`, or by using the normal dashboard, or the `.deep.json`, check that:
- Is there one Patient subject? Does it have the right `fullIdentifier`?
- Is there one Visit subject? Does it have the right `parents`? Does it have the right `fullIdentifier`?
- Do forms have the correct `subject` and `relatedSubjects`?
- Are the Patient Information and Visit Information forms marked as complete?
- Are all the other data forms marked as Incomplete, but not Invalid? (Except for AUDIT-C which is known to be buggy)
- Do all the forms have the correct `relatedSubjects`?
